skimage/transform/_warps.py

```diff
@@ -898,3 +898,146 @@ def coord_map(*args):
     _clip_warp_output(image, warped, order, mode, cval, clip)
 
     return warped
+
+
+def _linear_polar_mapping(output_coords, k_ang, k_rad, center):
```

Comment:
https://github.com/scikit-image/scikit-image/blob/f998c6334501d4724ded875aea04cd8bef4c6c6a/skimage/color/colorconv.py#L1508

Reply:
good catch! Here it's a private function though and I am hesitant to change that unless it's too much redundancy

Reply:
Let's keep this for a future issue.